src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java

```diff
@@ -346,9 +365,28 @@ public void recalculateMetrics() {
         updateMultiGaugeIntegerForMinuteRanges(releaseExamGauge, examRepository::countExamsWithStartDateBetween);
         updateMultiGaugeIntegerForMinuteRanges(releaseExamStudentMultiplierGauge, examRepository::countExamUsersInExamsWithStartDateBetween);
 
+        if (localCIBuildJobQueueServiceOptional.isPresent()) {
+            updateLocalCIMetrics();
+        }
```

Comment:
Why exactly is this only updated every 5 minutes? This is data that changes constantly even within one minute, so I think always returning up to date information here would be important.

I'd say that you should ideally return fresh data with every prometheus request (so if you need to schedule it every 15s)

Reply:
I agree, this should update without a schedule like "normal" metrics